benchmarks/experiment_runner.py

```diff
@@ -232,6 +233,21 @@ def prepare_inputs(self, example_inputs, should_randomize_input):
       inputs_list.append(inputs)
     return inputs_list
 
+  def dump_profile_info(self, prof, model_name):
+    assert prof is not None, 'Expecting profiler to be defined!'
+    if not self._args.profile_cuda_dump:
+      logger.warning('Profling enabled, but dumping tracing/kernel summary disabled.') 
```

Comment:
s/Profling/Profiling/

Reply:
Oh I can do suggestions here as well (Sorry I'm new to reviewing on Github):
```suggestion
      logger.warning('Profiling enabled, but dumping tracing/kernel summary disabled.') 
```